src/backend/distributed/Makefile

```diff
@@ -5,7 +5,7 @@ citus_top_builddir = ../../..
 
 MODULE_big = citus
 EXTENSION = citus
-EXTVERSIONS = 5.0 5.0-1 5.0-2
+EXTVERSIONS = 5.0 5.0-1 5.0-2  5.1-1
```

Comment:
It was intentional, but I'm fine with either killing it or adding a "newline" (it's not one due to the \\). Unless you protest I'll do the latter.

No, EXTVERSIONS doesn't have any "builtin" meaning, it's just used to generate DATA_built.  I was wondering about also generating the repetitive `$(EXTENSION)--5.0-2.sql` from it somehow; but thought it not to be urgent at this point.